cli/cmd/stat.go

```diff
@@ -452,7 +452,7 @@ func printStatJSON(statTables map[string]map[string]*row, w *tabwriter.Writer, o
 					entry.LatencyMSp99 = &stats[key].latencyP99
 					entry.TLS = &stats[key].tlsPercent
 
-					if showTCPStats(options, resourceType) {
+					if resourceType != k8s.Authority {
```

Comment:
i like that we're no longer passing `statOptions` into this function, but it does mean the logic to determine whether to print TCP stats lives in 2 places. i'd prefer to continue calling `showTCPStats` here, as it takes into account `outputFormat` already.